iot2000.coffee

```diff
@@ -0,0 +1,35 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	version: 1
+	slug: 'IOT2000'
+	aliases: [ 'IOT2020', 'IOT2040' ]
+	name: 'Siemens IOT2000'
+	arch: 'i386'
+	state: 'preview'
```

Comment:
I believe the state is "experimental" ?

Reply:
This still shows the wrong state. Are you sure the comments have been addressed @telphan ?

Reply:
Ping @telphan

Reply:
@floion @agherzan I am waiting on the supervisor to merged, the changes are made, just not pushed.